src/cli/commands/load.js

```diff
@@ -58,30 +67,46 @@ export default class PluginGraphCommand extends Command {
 
   async run() {
     const {
-      args,
+      argv,
       flags: {
+        output: defaultOutput,
         "github-token": githubToken,
         "sourcecred-directory": basedir,
         "max-old-space-size": maxOldSpaceSize,
         plugin,
       },
     } = this.parse(PluginGraphCommand);
-    const repo = stringToRepo(args.repo);
+    const repos = argv.map((s) => stringToRepo(s));
+    const outputRepo =
```

Comment:
I dislike nested ternary expressions, as I find them hard to read. (And nested ternary expressions with IIFEs are right out.)

Reply:
Changed, but for the record I think that nested conditional expressions
get way more hate than they deserve. :-)